### PR TITLE
chore(phrases,test): update M2M RBAC i18n content

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

update M2M RBAC i18n content

## Testing

N/A

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/phrases/src/locales/en/translation/admin-console/user-details.ts [hunk 1]
```diff
@@ -64,7 +64,8 @@ const user_details = {
   roles: {
     name_column: 'Role',
     description_column: 'Description',
-    assign_button: 'Assign Roles',
+    /** UNTRANSLATED */
```
Comment (simeng-li) [inline packages/phrases/src/locales/en/translation/admin-console/user-details.ts:67]: en untranslated?

File: packages/phrases/src/locales/it/translation/admin-console/user-details.ts [hunk 2]
```diff
@@ -2,7 +2,7 @@ const user_details = {
   page_title: 'Dettagli utente',
   back_to_users: 'Torna alla gestione utenti',
   created_title: 'Questo utente è stato creato con successo',
-  created_guide: 'Ecco le informazioni per aiutare l’utente con il processo di accesso.',
+  created_guide: "Ecco le informazioni per aiutare l'utente con il processo di accesso.",
```
Comment (simeng-li) [inline packages/phrases/src/locales/it/translation/admin-console/user-details.ts:5]: ‘ => " by design?

Comment (darcyYe) [reply packages/phrases/src/locales/it/translation/admin-console/user-details.ts:5]: There is a single quote mark in the sentence, so use double quotes is expected.

File: packages/phrases/src/locales/en/translation/admin-console/user-details.ts (hunk 1 above)
Comment (darcyYe) [reply packages/phrases/src/locales/en/translation/admin-console/user-details.ts]: Removed.

File: packages/phrases/src/locales/it/translation/admin-console/user-details.ts (hunk 2 above)
Comment (simeng-li) [reply packages/phrases/src/locales/it/translation/admin-console/user-details.ts:5]: ok, just notice that.

**Review by simeng-li — APPROVED** [review]
LGTM

**Pull request merged**